sample-tracking-status-overview-domain/src/main/groovy/life/qbic/business/samples/download/ComposeManifest.groovy

```diff
@@ -0,0 +1,37 @@
+package life.qbic.business.samples.download
+
+/**
+ * <b>Compose a download manifest based on the output of {@link DownloadSamplesOutput}</b>
+ *
+ * <p>This class composes a download manifest based on the output of {@link DownloadSamplesOutput}.</p>
+ *
+ * @since 1.0.0
+ */
+class ComposeManifest implements DownloadSamplesOutput{
```

Comment:
Why is this in this branch?
Might it be, that you branched of the other feature branch instead of development?

Reply:
@wow-such-code